### ISSUE TITLE
Better `forc-documenter` CI error reporting

### ISSUE THREAD

**Issue by binggh** [FuelLabs/sway#1489]
The current output for the [forc-documenter-dry-run](https://github.com/FuelLabs/sway/runs/6319183058?check_suite_focus=true) job is not very informative about what went wrong with the docs:

```
/home/runner/.cargo/bin/cargo run --bin forc-documenter write-docs --dry-run
   Compiling forc-documenter v0.0.0 (/home/runner/work/sway/sway/scripts/forc-documenter)
    Finished dev [unoptimized + debuginfo] target(s) in 2.27s
     Running `target/debug/forc-documenter write-docs --dry-run`
Running forc --help using forc 0.[12](https://github.com/FuelLabs/sway/runs/6319183058?check_suite_focus=true#step:6:12).1

forc addr2line: documentation ok.
forc build: documentation ok.
forc clean: documentation ok.
Error: Documentation inconsistent for forc completions - please run `cargo run --bin forc-documenter write-docs`. If you have made local changes to any forc native commands, please install forc from path first: `cargo install --path ./forc`, then run the command.
Error: The process '/home/runner/.cargo/bin/cargo' failed with exit code 1
```

Two nice improvements I can identify at the moment:

- [ ] print the diff so that we know exactly what is inconsistent without having to manually run the script ourselves
- [ ] do not stop at the first failed command, rather finish iterating all commands and report the results

**Comment by binggh** [comment]
This issue is no longer relevant after #1545 was merged - `forc-documenter` is now an mdbook preprocessor which runs during the `mdbook build` CI job and automatically applies forc command changes into the book instead of forcing us to run `forc-documenter` manually to write the updated docs within the book. Therefore the diff feedback is no longer necessary.
